### ISSUE TITLE
for loop *list with many params generate error code?

### ISSUE THREAD

**Issue by thesolitudeofnoblesoul** [pigpigyyy/Yuescript#49]
Realy stupid code EX:
```moonscript
for name, members in *modules
  print name, member
```
Expect stupid code:
```lua
local _list_0 = modules
for _index_0 = 1, #_list_0 do
  local name, members = _list_0[_index_0]
  print(name, member)
end
```
But generated worng code:
```lua
local _list_0 = modules
for _index_0 = 1, #_list_0 do
  local namemembers = _list_0[_index_0]
  print(name, member)
end
```

I thinks you should check it out!
Thanks and regards!

**Comment by pigpigyyy** [comment]
Sorry for missed this issue for some days! Should now be fixed by commit ea82666506b57d6e905b7f2e5fe78498fe5a7abd.

**Comment by thesolitudeofnoblesoul** [comment]
Confirm Fixed!
Thanks!
